### ISSUE TITLE
Dockerfile python version error

### ISSUE THREAD

**Issue by thatreguy** [JonathanSalwan/Tigress_protection#17]
Could NOT find PythonInterp: Found unsuitable version "2.7.12", but
required is at least "3.6" (found /usr/bin/python)
![Screenshot 2021-08-09 163324](https://user-images.githubusercontent.com/44264798/128696993-93e0bcd5-45ca-437c-b203-e4c7ce99ad27.png)

**Comment by thatreguy** [comment]
After struggling for a while and changing python version to 3.6 in the Dockerfile, I'm getting another error. Now while building Triton it complains about missing python 2.7(The pin tracer needs Python 2.7)

**Comment by JonathanSalwan** [comment]
Yeah Pin is only available with Python 2.7. If you want to compile without the Pin tracer, remove the `-DPINTOOL` option from the cmake command line.
